satellite/satellitedb/repairqueue.go

```diff
@@ -59,7 +57,7 @@ func (r *repairQueue) sqliteSelect(ctx context.Context) (seg *pb.InjuredSegment,
 		if err != nil {
 			return err
 		}
-		res, err := tx.Tx.ExecContext(ctx, r.db.Rebind(`UPDATE injuredsegments SET attempted = ? WHERE path = ?`), time.Now().UTC(), path)
+		res, err := tx.Tx.ExecContext(ctx, r.db.Rebind(`UPDATE injuredsegments SET attempted = datetime('now') WHERE path = ?`), path)
```

Comment:
Should we use `timezone('utc', now())` here as well?

Reply:
sqlite supposed uses UTC by default

Reply:
sqlite doesn't have the `timezone()` function, and doesn't understand the standard `AT TIME ZONE` syntax either. Bill is right, though: sqlite3 will use UTC. `datetime('now')` should be fine. And even if it wasn't, there isn't much risk of a sqlite3 database being used by multiple clients with different local time zones set.